.env.example

```diff
@@ -129,6 +129,7 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 ## Secret storage
 # SECRET_STORAGE_BACKEND= # optional, should be one of: SecretStorage::DbBackend (default) or SecretStorage::HashicorpVault
 # SECRET_STORAGE_SHARING_GRANTS=true # optional, instead of sharing global secrets by default, access has to be granted
+# SECRET_STORAGE_DEPRECATED_IDS=a/b/c/d # optional, ids that should longer be resolved, to not use a id on a new deploy but keep the old ones working
```

Comment:
this seems like a strange place to have this as it'll require a re-deploy if someone wants to deprecate a key.  Can't really think of a better place though, unless we store it in db or something?